### ISSUE TITLE
Tests through PhantomJS fail with strange error

### ISSUE THREAD

**Issue by lookfirst** [systemjs/systemjs#382]
This was working fine before jspm 0.15. =)

https://travis-ci.org/lookfirst/systemjs-seed/jobs/56042263

```
./node_modules/gulp-helpers/node_modules/karma/bin/karma start --single-run --browsers PhantomJS
INFO [karma]: Karma v0.12.31 server started at http://localhost:9876/
INFO [launcher]: Starting browser PhantomJS
INFO [PhantomJS 1.9.8 (Mac OS X)]: Connected on socket XNI4sDAUusOiLfsV9WM5 with id 37792976
PhantomJS 1.9.8 (Mac OS X) ERROR
  TypeError: 'undefined' is not a function (evaluating 'a.bind(e)')
  at /Users/jon/checkout/systemjs-test/systemjs-seed/jspm_packages/system.js:1

ERROR [PhantomJS 1.9.8 (Mac OS X)]: TypeError: 'undefined' is not a function (evaluating 'a.bind(e)')
at http://localhost:9876/base/jspm_packages/system.js?08662bb9df157e473710cdca2595fc83c4534360:1
```

**Comment by guybedford** [comment]
Thanks, fixed in https://github.com/systemjs/systemjs/commit/781b4787bc848baf0006757ba7f968e8f61fe2ef.

**Comment by lookfirst** [comment]
Sweet! Can we get a fast release for this with jspm too?

Also, shouldn't there be unit tests for this stuff?

**Comment by guybedford** [comment]
You can use `jspm dl-loader --edge`.

**Comment by guybedford** [comment]
Fixed in 0.16.6.
